Sources/StreamChat/Models/ChatMessage.swift

```diff
@@ -316,7 +316,7 @@ public extension ChatMessage {
 
     /// Returns the attachments of `.file` type.
     ///
-    /// - Important: The `fileAttachments` are loaded lazily and cached to maintain high performance.
+    /// - Important: The `unknownAttachments` are loaded lazily and cached to maintain high performance.
```

Comment:
Hmm, why is this changed, the file attachments are declared here?

Reply:
Nice catch 👍 Fixed it